packages/appium/lib/cli/parser.js

```diff
@@ -98,6 +98,9 @@ function addExtensionsToParser (sharedArgs, subParsers, debug = false) {
        help: `Uninstall a ${type}`},
       {command: 'update', args: extensionArgs[type].update,
        help: `Update installed ${type}s to the latest version`},
+      {command: 'run', args: extensionArgs[type].run,
+       help: `Run a script (cached inside the ${type}s package.json under the ` +
```

Comment:
I would say `defined` rather than `cached`. I'm not sure what cached would mean here.

Reply:
also, `${type}'s`